ui/pom.xml

```diff
@@ -86,7 +86,7 @@
               <goal>npm</goal>
             </goals>
             <configuration>
-              <arguments>install --no-optional -g angular-cli</arguments>
+              <arguments>install --no-optional -g @angular/cli</arguments>
```

Comment:
Is the npm that we install (from Node 6) already new enough to understand scoped packages ? (afaik available since Node 2.7.0)

Actually I would like to update Node to version 8, too, but it seems that this automatic installation for node from within the build does not work on Circle (leads to an endless loop with permission errors0.

Reply:
I has worked for me our karma-xvfb image, that has Node 6 installed. But of course it can wait until to be done with upgrade to Node 8.

Reply:
np, was just asking ;-)